src/libraries/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSourceActivity.cs

```diff
@@ -150,9 +150,13 @@ public enum ActivityIdFormat
     }
     public sealed class ActivitySource : IDisposable
     {
+        public ActivitySource(string name) { throw null; }
+        [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
         public ActivitySource(string name, string? version = "") { throw null; }
+        public ActivitySource(string name, string? version = "", System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, object?>>? tags = default) { throw null; }
```

Comment:
would we need to do the `[System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]` on this one too when we support schemaUrl as a parameter? (#63651)

Reply:
Yes, when adding a new constructor with extra optional parameter, we can add the `[System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]` to the existing one. 

I was thinking in the future if we add more options, we may start having `ActivitySourceOptions` and a constructor take this options object.